core/client/src/main/java/alluxio/client/file/options/CreateFileOptions.java

```diff
@@ -113,11 +125,19 @@ public Mode getMode() {
   /**
    * @return the under storage type
    */
+  @JsonIgnore
```

Comment:
If `WriteType` supports `getUnderStorageType` and we are exposing `WriteType`, we do not need to have separate getters for the Alluxio/UnderStorage type